### DISCUSSION TITLE
Error opening translation file English

### DISCUSSION THREAD

**Discussion by wyatt-wong** [mamedev/discussions#141] [Q&A] [answered]
I have successfully built MAME 0.269 for macOS from the source. When I run it in Applications folder, why would I receive "Error opening translation file English" ?

![Screenshot 2024-09-16 at 9 43 11 AM](https://github.com/user-attachments/assets/729f7343-cc49-4977-bfc7-8e25736cc231)

Furthermore, I found the screen will first turns into BLACK screen and then returns to normal, and then it will show the MAME screen. Why would MAME turns my Mac screen into BLACK screen first ?

![Screenshot 2024-09-16 at 9 39 06 AM](https://github.com/user-attachments/assets/670a59a3-0eab-476b-8197-6eb26b75382a)

**Comment by wyatt-wong** [comment]
Is there any update on this issue ?

**Comment by cuavas** [comment, marked as answer]
It’s because, by default, MAME searches paths relative to the working directory.  Assuming you haven’t changed the setting, it’s looking for the `language` folder in the working directory, but it isn’t there.

Either configure MAME to use absolute search paths for everything, or start MAME from the directory where the support files are.
